src/lib/api/supabase.ts

```diff
@@ -0,0 +1,14 @@
+import { createClient } from '@supabase/supabase-js'
+import { Database } from 'types/database.types'
+
+export const sudoPublicDbClient = createClient<Database>(
```

Comment:
**suggestion** i think this might be better suited to a `lib/supabase.ts` wdyt?

Reply:
idm. I think we need a complete overhaul on our file structure as it is